api/v1/controllers/samples.js

```diff
@@ -57,21 +93,33 @@ module.exports = {
    * @param {Function} next - The next middleware function in the stack
    */
   findSamples(req, res, next) {
+    // Check if Cache is on for Wildcard Sample query
+    if (featureToggles.isFeatureEnabled('cacheGetSamplesWildcard')) {
+      const query = req.query.name;
+      helper.cacheEnabled =
+        query ? (query.indexOf('*') > -1) ? true : false : false;
```

Comment:
I had to read this line multiple times to understand it. Please use extra parentheses to make it more readable, or break into two statements.

Reply:
Got it 👍